content.md

```diff
@@ -1,125 +1,126 @@
+    - When onboarding the main CDN dev team onto the a shared rotation with SREs, I documented oncall expectations in line with the standards I created above. 
+- **Provided technical solutions to reliability problems.** 
+  - Moved configuration integration testing into its own service, independent from code testing, resulting in a more stable and consistent build/certification pipeline. 
+  - **Designed solution for a major reliability problem** in which a single node failures caused delays and buffering across all videos and large files delivered. Worked with the CDN Devs to implement this solution to identify and route around failed nodes even before they were removed from the working pool. 
+ 
+ 
+### Google, Inc. - **Site Reliability Engineer** - *February 2008 - February 2020* 
+ 
+During my tenure as an SRE, I've built and run Google enterprise infrastructure using open source tools on vanilla-ish Linux, 
+proprietary third-party platforms, Google's production infrastructure, and Google Cloud Platform; I've carried oncall pagers 
+with 5-minute and 30-minute response SLAs; and I've built tools and processes to reduce and automate away toil. 
+ 
+#### Backups Tech Lead - Enterprise Data Storage - *(2017-Present)* 
+- **Designed Google’s next gen internal backup tool** from the ground up. 
+  - Built on **Google Cloud Platform** to manage provisioning of storage, storage class migration, data retention, authentication/authorization, key escrow, and encryption. 
+  - Made setup and restore **self-service**, leading to happier users and less toil on the team. 
+- **Set technical direction** for a team of between **2 and 6 engineers** to build, test, and roll out our new service. 
+- **Managed coordination with other teams** to: 
+  - **Push for upstream features** in Google Cloud Storage by speccing requirements, coordinating changes, and finding headcount from internal initiatives. 
+  - **Capture user requirements** by working closely with alpha-/beta-testers to capture pain points, hard requirements, and wishlist features. 
+  - **Conform to security best practices**, seeking input from security team and building out features as needed to help users avoid security pitfalls. 
+ 
+#### Enterprise Data Storage - Individual Contributor - *(2014 - Present)* 
+- **Migrated XX,000 long-term storage images** from deprecated tape system onto cloud storage, including integrity checking and modifying our backups management software. 
```

Comment:
Is XX intentional or a placeholder?

Reply:
Intentional for NDA reasons. (Also I now do not remember the number.) I can say "tens of thousands" to make it clearer?